_posts/2023-17-10-remote-attestation-blog-part1.markdown

```diff
@@ -0,0 +1,138 @@
+
+This framework allows changes to the user’s computer, the attested (your remote servers), to be detected by authorized parties, the attestors (you on your local servers). In this framework, the attestor machine is already trusted by the customers, while the nodes they are provisioning through the cloud service providers are not. 
+
+Remote attestation can provide different services, such as measured boot attestation and runtime integrity monitoring, using a hardware-based cryptographic "Root of Trust" (RoT), for example, a Trusted Platform Module (TPM). A TPM is a chip that conforms to the standards of a secure cryptoprocessor, which is a dedicated microcontroller designed to secure hardware through integrated cryptographic keys. It is used to store artifacts that work to authenticate the platform it is on. 
+
+
+<div style="text-align: center;">
+<img  src="../assets/images/RemoteAttestation.png" alt="Remote Attestation Diagram">
+</div>
+
+**Figure 1: Remote Attestation is used to provide trust between an untrusted party and a trusted party. You can use services like Measured Boot Attestation and Runtime Integrity Monitoring to prove systems are trustworthy.**
+
+In this article, we will detail the different aspects of remote attestation based on the following sections:
+<ul>
+    <li> What is Secure Boot? </li>
+    <li> What is Measured Boot? </li>
+    <li> Measured Boot and Secure Boot go Hand in Hand </li>
+    <li> Integrity Measurement Architecture (IMA) for Continuous Attestation</li>
+    <li> Introduction to Keylime, A Remote Attestation Software </li>
+</ul>
+
+<h3 style="text-align left; color:#6B98BF;"> What is Secure Boot? </h3>
+
+Unified Extended Firmware Interface (UEFI) Secure Boot is a security measure developed to ensure that a device is booted using only software trusted by the Original Equipment Manufacturer (OEM). The goal is to prevent malicious software from being loaded and executed early in the boot process. During secure boot, the next component is verified by the component before.
+
+Secure Boot prevents boot if the signature cannot be validated against a certificate enrolled in the bootchain. For example if someone changes the kernel to a custom one, it is not signed by the OS vendor. This causes the boot to stop because the bootloader couldn't verify the signature. The process of checking the kernel's signatures occurs before the OS ever runs.
+
+<h3 style="text-align left; color:#6B98BF;"> What is Measured Boot? </h3>
```

Comment:
Very minor issue but it seems that because there's not an empty line between the H3 (line 49) and the opening paragraph of the section (line 50), at least on GH, line 50 is not interpreted as a paragraph and does not get wrapped in `<p>` tags. Not sure if the markdown parser used to render the blog posts acts differently though

Reply:
I added a space, so hopefully that clears things up.